TeleSharp.Generator/TL2JSON.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace TeleSharp.Generator
+{
+    static class TL2JSON
```

Comment:
This class is the heart of TL language translation. I wrote it using Regex, although I had no experience in Regex.